catalogue/webapp/components/IIIFViewer/ViewerBottomBar.tsx

```diff
@@ -1,3 +1,5 @@
+// https://stackoverflow.com/questions/25993861/how-do-i-get-typescript-to-stop-complaining-about-functions-it-doesnt-know-abou
+/* eslint-disable dot-notation */
```

Comment:
This addresses lines like https://github.com/wellcomecollection/wellcomecollection.org/pull/9667/files#diff-6f4e90f31337f07e6506a070a2a7794c1c1534f7cc510db37ae1deeed69e7fd2R113, see link for more details